packages/code/src/Code.tsx

```diff
@@ -382,55 +352,36 @@ function Code({
     </div>
   );
 
-  const onScroll: React.UIEventHandler<HTMLDivElement | HTMLPreElement> = e => {
-    e.persist();
-    debounceScroll(e);
-  };
+  return (
+    <div className={wrapperStyle}>
+      {showWindowChrome && (
+        <WindowChrome chromeTitle={chromeTitle} darkMode={darkMode} />
+      )}
 
-  const commonWrapperProps = {
-    chromeTitle,
-    darkMode,
-    showWindowChrome,
-  } as const;
-
-  if (!multiline) {
-    return (
-      <CodeOuterWrapper {...commonWrapperProps}>
-        <div
-          {...(rest as DetailedElementProps<HTMLDivElement>)}
+      <div
+        className={css`
+          display: flex;
+        `}
+      >
+        <pre
+          {...(rest as DetailedElementProps<HTMLPreElement>)}
           className={wrapperClassName}
           onScroll={onScroll}
-          ref={scrollableSingleLine}
+          ref={scrollableElement}
         >
           {renderedSyntaxComponent}
-        </div>
+        </pre>
 
         {copyBar}
```

Comment:
Seeing accessibility issues from the Icon in the copy button

Reply:
Definitely possible that they were present before, but if possible we should address while updating the component

Reply:
Not a regression, and is an issue with the IconButton component itself. Made a ticket: https://jira.mongodb.org/browse/PD-458